crm_deduplicate_acl/wizards/partner_merge.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# © 2016 Antiun Ingeniería S.L. - Jairo Llopis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import SUPERUSER_ID, api, models
+
+
+class BasePartnerMergeAutomaticWizard(models.TransientModel):
+    _inherit = "base.partner.merge.automatic.wizard"
+
+    @api.cr_uid_context
+    def _merge(self, cr, uid, partner_ids, dst_partner=None, context=None):
```

Comment:
[The problem is the missing model_cr decorator, that has already landed in master](https://github.com/odoo/odoo/blob/master/openerp/api.py#L435). For v8 I could do that, but it's not called from v8 methods anyway.